### PR TITLE
[dashboard] it makes more sense to have the green reflect the "remaining hours" instead of "cosumed". Having the "consumed" it creates confusion. see issue 9702

### PR DESCRIPTION
**Pull request by ionutale**
### What type of PR is this?
/kind bug

### What this PR does / why we need it:
UX improvements for plans remained hours

### Which issue(s) this PR fixes:
plan remaining hours and progress bar #9702

### CODE REVIEW

**Comment by roboquat** [comment]
@ionutale: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by ionutale** [comment]
Release Notes:

- Aligned the `remaining hours` text with the green color of the `progress bar`

**Comment by ionutale** [comment]
thankyou @gtsiolis

File: components/dashboard/src/settings/Plans.tsx [hunk 1]
```diff
@@ -665,10 +665,7 @@ export default function () {
                         </Tooltip>
                         {typeof accountStatement?.remainingHours === "number" &&
                         typeof currentPlan.hoursPerMonth === "number" ? (
-                            <progress
-                                value={currentPlan.hoursPerMonth - accountStatement.remainingHours}
-                                max={currentPlan.hoursPerMonth}
-                            />
+                            <progress value={accountStatement.remainingHours} max={currentPlan.hoursPerMonth} />
```
Comment (gtsiolis) [inline components/dashboard/src/settings/Plans.tsx:668]: **suggestion**: I was thinking we could a) make this more useful for the upcoming usage-based pricing, b) keep the existing approach, c) avoid the negative aspects that could be attached with the draining hours progress, d) improve accessibility on this area with removing the tooltip used here by making this look like the following:

| BEFORE  | AFTER |
|-|-|
| usage-before | usage-after |

What do you think?

**Review by gtsiolis — COMMENTED** [review]
Thanks for contributing, @ionutale! 🍊  

You'll also need to sign a Contributor License Agreement (CLA)[[1](https://en.wikipedia.org/wiki/Contributor_License_Agreement)] once before merging your first contribution. Looping in @meysholdt to reach out about the CLA. 🏓

Left one comment above, let me know what you think!

**Comment by gtsiolis** [comment]
/werft run with-payment

☝️ Trying this for the first time! Cc @jankeromnes

**Comment by ionutale** [comment]
@gtsiolis I see this PR that has not been merged yet. Do I need to do something else?

**Comment by meysholdt** [comment]
hi @ionutale! I've emailed our CLA your way. I believe that is still missing before we can merge this PR. Apologies for sending it so late. If you have questions, feel free to reach me via `moritz@gitpod.io`.

**Comment by ionutale** [comment]
> hi @ionutale! I've emailed our CLA your way. I believe that is still missing before we can merge this PR. Apologies for sending it so late. If you have questions, feel free to reach me via `moritz@gitpod.io`.

hi @meysholdt , I've signed the CLA

**Comment by jankeromnes** [comment]
Hey, thanks for improving Gitpod's dashboard, and for signing the CLA!

Let's try to get this merged.

- Firstly, the preview environment is gone, so I'll trigger a new build
- Secondly, let's see if we can find a good compromise on the "remaining hours" (simple 👍 but counting down could be a bit stressful) vs "workspace hours usage since (date) (numbers)" (much too long and complicated for my taste 😅) -- I'll reply on the review comments directly

/werft run

**Comment by jankeromnes** [comment]
@ionutale Apologies, it looks like we broke the Gitpod build, and you don't have the fix yet in your Pull Request / branch.

In order to fix the build, could you please rebase this Pull Request?

If it helps, you could achieve this like so:
1. Open this PR in Gitpod: https://gitpod.io/#https://github.com/gitpod-io/gitpod/pull/9716
2. Run `git pull --rebase upstream main` to rebase
3. Run `git push -f origin plan-remaining-hours-and-9702` to update this PR

**Comment by ionutale** [comment]
hi @jankeromnes,
followed your instructions.
please have a look at it

**Comment by gtsiolis** [comment]
@ionutale Did you force pushed your changes?  I see there are not new changes after the message from @jankeromnes in https://github.com/gitpod-io/gitpod/pull/9716#issuecomment-1150000506.

**Comment by ionutale** [comment]
the force was not working, so I did it manually from the github frontend

**Comment by laushinka** [comment]
@gtsiolis Reading @jankeromnes's [comment](https://github.com/gitpod-io/gitpod/pull/9716#issuecomment-1149961811), do we still need alignment on the wording?

**Comment by gtsiolis** [comment]
@laushinka Overall, I agree with @jankeromnes in https://github.com/gitpod-io/gitpod/pull/9716#is suecomment-1149961811. The proposal in https://github.com/gitpod-io/gitpod/pull/9716#discussion_r864044339

**Comment by gtsiolis** [comment]
Thanks, @ionutale! Could you make the changes according to the designs in https://github.com/gitpod-io/gitpod/pull/9716#issuecomment-1152475465?

For the element on the bottom, you could also experiment with adding a component variable for the **[Pill Label](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/PillLabel.tsx)** component, but it should be completely fine to skip this and ad-hoc style this element using the utility classes available.

Using the following classes could help:

```css
/* 1st element */
font-semibold text-gray-500 dark:text-gray-300

/* 2nd element */
font-medium text-sm text-gray-500 bg-gray-100 dark:text-gray-400 dark:bg-gray-800 rounded-xl p-1 px-2 text-sm

/* 2nd element, dimmed part */
text-gray-400 dark:text-gray-500

```

**Comment by ionutale** [comment]
give me a few days, and I will come back to you

**Pull request closed without merging**